### DISCUSSION TITLE
Any way to list live followed Twitch streamers via CLI?

### DISCUSSION THREAD

**Discussion by phxn** [streamlink/streamlink#4782] [Q&A] [answered]
Been using Streamlink CLI to watch Twitch streams, I was wondering whether there was any way to show a list of live streamers I follow?

This is so I can cut out going to Twitch website completely and just browse via PowerShell.

Thanks!

**Comment by bastimeyer** [comment, marked as answer]
Streamlink is not a tool for browsing streams. You'll have to write your own wrapper for that and access the Twitch API yourself.
https://dev.twitch.tv/docs/api

Alernatively, take a look at existing browsers for Streamlink:
https://streamlink.github.io/applications.html
https://streamlink.github.io/thirdparty.html

**Reply by phxn** [reply]
I understand, just wondered whether there was a plugin or command I was missing.

It seems there is nothing that will do what I want.
